polaris-ui/src/app/Routes.tsx

```diff
@@ -13,9 +13,24 @@ import CaseSearchResults, {
 import Case, {
   path as casePath,
 } from "./features/cases/presentation/case-details";
+import { testAppInsightsConnection } from "../app/common/utils/appInsightsUtils";
 
 export const Routes: FC = () => {
   const { state } = useLocation();
+  const [isAppInsightActive, setIsAppInsightActive] = useState(true);
+  useEffect(() => {
+    if (process.env.NODE_ENV === "production") {
+      setTimeout(async () => {
+        if (!(await testAppInsightsConnection())) {
```

Comment:
this timeout is added , in local development  when testing the appinsight test  post preflight call triggered by us is been blocked by CORS issue, but browser is good once the track post call is made by appInisight itself. So the timeout is the time for the app to make its first appinisght call. As i understand this will be a problem only for our local development. In all the production environment we got the proxy for the appInsight post url. So it shouldn't be an issue (but we can only test it by deploying to dev.)